src/volttron/client/commands/control.py

```diff
@@ -145,13 +145,13 @@ def escape(pattern):
 
 
 def filter_agents(agents, patterns, opts):
```

Comment:
Please document the filter_agents method to have the different options be called out in the docs.